woocommerce/woocommerce-ios
# Display shipping tax value in order total bottom pane

Closes: #11960


## Description

- Display shipping tax value in order total bottom pane

## Testing instructions

- Open Orders tab
- Tap + button to start adding an order
- Add some products
- Check that in the bottom page shipping tax is not showed
- Add shipping amount
- Check that in the bottom page shipping tax is now showed

## Screenshots


| Without shipping tax      | With shipping tax |
| ----------- | ----------- |
| ![Simulator Screenshot - iPhone 15 Pro - 2024-02-14 at 14 39 57](https://github.com/woocommerce/woocommerce-ios/assets/6242034/675f1a4e-cea1-4c41-ace0-7c7a7bf53032)      | ![Simulator Screenshot - iPhone 15 Pro - 2024-02-14 at 14 40 09](https://github.com/woocommerce/woocommerce-ios/assets/6242034/e9c43573-adce-4a13-b164-7064eb80c5d5)       |

---
